.github/workflows/deploy.yaml

```diff
@@ -386,13 +392,37 @@ jobs:
           deployment/seed-data.sh
         working-directory: .
 
+      - uses: actions/download-artifact@v3
+        with:
+          name: gaiaxkey
+          path: deployment/terraform/participant
+
+      - name: 'Push region verifiable credential'
+        run: |
+          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.identityhub:identity-hub-cli:$cli_version:jar:all -DoutputDirectory=.
+          
+          id=$(uuidgen)
+          
+          java -jar identity-hub-cli-$cli_version-all.jar -s=$identityHubUrl vc add \
+            -c='{"id":"'$id'","credentialSubject":{"region":"'$region'"}}' \
+            -b="did:web:$DID_HOST" \
```

Comment:
What does -b means?

Reply:
It's the "subject" field. The CLI brings descriptions for all fields in case you want to have a look: https://github.com/agera-edc/IdentityHub/blob/main/client-cli/src/main/java/org/eclipse/dataspaceconnector/identityhub/cli/AddVerifiableCredentialCommand.java#L47

Reply:
Why did you use b instead of s?

Reply:
"-s" was taken for "server": https://github.com/agera-edc/IdentityHub/blob/main/client-cli/src/main/java/org/eclipse/dataspaceconnector/identityhub/cli/IdentityHubCli.java#L33

I kept "-s" because its the same name as in the Registration Service.